telegram/_bot.py

```diff
@@ -965,6 +982,8 @@ async def send_photo(
                 receive a notification with no sound.
             protect_content (:obj:`bool`, optional): Protects the contents of the sent message from
                 forwarding and saving.
+            message_thread_id (:obj:`int`, optional): Unique identifier for the target message
+                thread (topic) of the forum; for forum supergroups only.
 
                 .. versionadded:: 13.10
```

Comment:
you sneaked the message_thread_id into the old versionadded

Reply:
D'oh. Thanks for spotting!